ccf_transaction_processor/scripts/start_cchost.sh

```diff
@@ -28,20 +28,11 @@ if [ $SGX_MODE == "SIM" ]; then
    CCHOST=${F_SERVICEHOME}/bin/cchost.virtual
 fi
 
-if [ -f ${F_SERVICEHOME}/run/cchost.pid ]; then
-    if ps -p $(cat ${F_SERVICEHOME}/run/cchost.pid) > /dev/null
-    then
-        yell cchost appears to be running already
-        exit -1
-    fi
-fi
-
```

Comment:
why are we dropping this?

Reply:
Thank you, the same check is performed in start_ccf_network.sh as well. So this is redundant, given that start_cchost is invoked only from either start_ccf_network or add_node_to_ccf. there is a check inside add_node_to_ccf as well. (Both these are wrappers for start_cchost which in addition to starting cchost does some necessary configuration as well)

Reply:
but we'll never call start_ccf_network.sh.

Reply:
i expect the ccf operator/member to only use the start_ccf_network or add_node_to_ccf script based on what the operator wants to do. This is independent of whether we deploy on same machine or multiple machines. cchost is never expected to be called by itself. since even after calling cchost, multiple other commands need to be invoked like configure. all of these are packaged inside the two wrappers based on if it is start or add